app/components/Form/ProjectFundingAgreementFormSummary.tsx

```diff
@@ -64,10 +60,19 @@ const ProjectFundingAgreementFormSummary: React.FC<Props> = ({
               netPaymentsToDate
               grossPaymentsToDate
               calculatedTotalPaymentAmountToDate
+              calculatedProponentsSharePercentage
+              totalProjectValue
               isPristine
```

Comment:
#1536 deals with additional funding sources so I haven't included anticipated funding diffs in this PR